schema.graphql

```diff
@@ -10,6 +10,19 @@ type Announcement {
 
 scalar DateTime
 
+type Employee {
+  displayName: String!
+  id: ID!
+  isPhotoUploaded: String!
+  photoUrl: String!
+}
+
+type EmployeeEventCollection {
+  anniversaryEmployees: [Employee!]!
+  birthdayEmployees: [Employee!]!
+  timeOffEmployees: [Employee!]!
```

Comment:
I am curious about  [Employee!] ! , what does it refer to ?

Reply:
It is the data type that the array must consist of! For ex all of the elements in `anniversaryEmployees` must be of type `Employee`. The type `Employee` then has to have certain fields like `display_name, birthday, etc.`